### PR TITLE
Support passthrough columns

### PR DESCRIPTION
**Pull request by jperez999**
This will allow the workflow object to recognize passthrough columns and update metadata of output files accordingly.

### CODE REVIEW

File: tests/unit/test_io.py [hunk 1]
```diff
@@ -259,10 +263,22 @@ def test_mulifile_parquet(tmpdir, dataset, df, engine, num_io_threads, nfiles, s
...
-        df_check[columns].sort_values(["x", "y"]),
-        df[columns].sort_values(["x", "y"]),
+        df_check[cont_names + label_names + passthru_names].sort_values(["x", "y"]),
+        df[cont_names + label_names + passthru_names].sort_values(["x", "y"]),
         check_index=False,
     )
+    # make sure out columns has all columns from start
+    assert all(col in df_check.columns.tolist() for col in columns)
+
+    if shuffle:
```
[the remark this answers is not in the document]
Comment (jperez999) [reply tests/unit/test_io.py:273]: To verify no column is dropped/ignored within the pipeline. Also added an operator to the test to make sure it goes through a phase execution cycle.

File: nvtabular/io/dask.py [hunk 2]
```diff
@@ -56,7 +57,9 @@ def _write_output_partition(
                 bytes_io=(shuffle == Shuffle.PER_WORKER),
                 num_threads=num_threads,
             )
-            writer.set_col_names(labels=label_names, cats=cat_names, conts=cont_names)
+            writer.set_col_names(
+                labels=label_names, cats=cat_names, conts=cont_names, passthru=passthru
```
[the remark this answers is not in the document]
Comment (jperez999) [reply nvtabular/io/dask.py:61]: This is a good point... @benfred  how should we handle this...

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Comment by benfred** [comment]
This PR isn't necessary anymore with the API changes.

**Pull request closed without merging**